### DISCUSSION TITLE
Deploying a custom build on a cluster

### DISCUSSION THREAD

**Discussion by vergills** [rook/rook#9313] [Q&A] [answered]
Hello all. 👋

I'm trying to deploy a custom build of Rook so I can see the changes I made in the codebase reflect in the cluster.

However, I'm not exactly sure how to do that.

I did a `make build` and it created a `/_output` directory with my custom Rook binary in a subfolder inside that, but I'm not sure how to properly deploy this and replace the general Rook binary that's running in the cluster with this one.

Any help or clues regarding this will be much appreciated, thanks! 🙌

**Comment by leseb** [comment, marked as answer]
After `make build` is done you should have a new container image named like `build-97814ef3/ceph-amd64`, the sha1 differs for each machine. Simply replace the image used by your Rook Operator with the newly created one.

**Reply by vergills** [reply]
That makes so much sense, thank you @leseb! 🎉
